### PR TITLE
Samsung Exynos AI LiteCore - Lower more ops and add tests to support LLM models

### PR DESCRIPTION
**Pull request by Jiseong-oh**
### Summary
Implemented lowering for additional ops and added tests to support LLM models.
* Add hardsigmoid, hardswish, hardtanh, layer_norm, maximum, prelu, pixel_unshuffle ops and testcase for supporting LLM.
and add axis infomation for rms norm.
* Replacing torch.randint(...).to(device) and torch.zeros(...).to(device) with creation calls that pass device=device

### Test plan
pytest backends/samsung/test/ops/test_hardsigmoid.py \
         backends/samsung/test/ops/test_hardswish.py \
         backends/samsung/test/ops/test_hardtanh.py \
         backends/samsung/test/ops/test_layer_norm.py \
         backends/samsung/test/ops/test_maximum.py \
         backends/samsung/test/ops/test_prelu.py\
         backends/samsung/test/ops/test_pixel_unshuffle.py -v

cc @SS-JIA @digantdesai @kimishpatel

### CODE REVIEW

File: backends/samsung/builders/op_pixel_unshuffle.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    def define_node(
+        self,
+        node: torch.fx.Node,
+        enn_graph: EnnGraph,
+        vals_to_ids: Dict[torch.Tensor, int],
+    ) -> bool:
+        input_id = self.define_tensor(node.args[0], enn_graph, vals_to_ids)
+
+        downscale_factor = cast(int, node.args[1])
```
Comment (psiddh) [inline backends/samsung/builders/op_pixel_unshuffle.py:35]: Could we validate the input rank before reporting this node as supported? `aten.pixel_unshuffle` accepts rank-3 and higher inputs, while ENN `SPACE_TO_DEPTH` may only support 4-D tensors. Since the support checker trusts the visitor return value, unsupported ranks could be delegated incorrectly. Please reject unsupported ranks or add tests confirming ENN supports them.

File: backends/samsung/test/ops/test_layer_norm.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            module,
+            inputs,
+            [gen_samsung_backend_compile_spec(TestConfig.chipset)],
+        )
+        (
+            tester.export()
+            .check_count({"torch.ops.aten.layer_norm.default": 1})
+            .to_edge_transform_and_lower()
+            .check_not(
+                ["executorch_exir_dialects_edge__ops_aten_native_layer_norm_default"]
```
Comment (psiddh) [inline backends/samsung/test/ops/test_layer_norm.py]: This checks `native_layer_norm`, but the op exported and preserved by this test is `layer_norm`. Could this check `executorch_exir_dialects_edge__ops_aten_layer_norm_default` instead, so it verifies that the op under test was delegated?

File: backends/samsung/builders/op_pixel_unshuffle.py (hunk 1 above)
Comment (Jiseong-oh) [reply backends/samsung/builders/op_pixel_unshuffle.py:35]: Thanks @psiddh it's good point. I updated it.

File: backends/samsung/test/ops/test_layer_norm.py (hunk 2 above)
Comment (Jiseong-oh) [reply backends/samsung/test/ops/test_layer_norm.py]: updated.

**Review by psiddh — APPROVED** [review]
